Public/Remove-PolarisRouteMiddleware.ps1

```diff
@@ -0,0 +1,41 @@
+<#
+.SYNOPSIS
+    Remove route middleware.
+.DESCRIPTION
+    Remove route middleware matching the specified name(s).
+.PARAMETER Name
+    Name of the middleware to remove.
+    Accepts multiple values and wildcards.
+    Accepts pipeline input.
+    Accepts pipeline input by property name.
+    Defaults to all names (*).
+.EXAMPLE
+    Remove-PolarisRouteMiddleware -Name JsonBodyParser
+.EXAMPLE
+    Remove-PolarisRouteMiddleware
+    Removes all route middleware.
+.EXAMPLE
+    Remove-PolarisRouteMiddleware -Name ParamCheck*, ParamVerify*
+    Removes any route middleware with names starting with ParamCheck or ParamVerify.
+.EXAMPLE
+    Get-PolarisRouteMiddleware | Remove-PolarisRouteMiddleware
+    Removes all route middleware.
+#>
+function Remove-PolarisRouteMiddleware {
```

Comment:
Nice to have: This function should take in a Polaris class object with a default value of $script:Polaris so that people can have multiple instances running in the same session.